source/clear-linux/tutorials/kernel-boot-params.rst

```diff
@@ -0,0 +1,45 @@
+.. _kernel_boot_params:
+
+Modify kernel boot parameters
+#############################
+
+Changing kernel boot paramters can drastically change the way the OS behaves,
+so be cautious when doing so. |CL| provides a simple way to do this that is
+consistent with its :ref:`stateless` nature. All configuration changes belong
+in the :file:`/etc/kernel/` path. The :command:`clr-boot-manager` generates
+boot entries, so we'll use it to run an update each time we change a boot
+paramter. 
+
```

Comment:
Documentation for the command line modification process is in the CONFIGURATION section of `man clr-boot-manager`